langchain4j-weaviate/src/test/java/dev/langchain4j/store/embedding/weaviate/LocalWeaviateEmbeddingStoreIT.java

```diff
@@ -1,30 +1,60 @@
 package dev.langchain4j.store.embedding.weaviate;
 
 import dev.langchain4j.data.segment.TextSegment;
+import static dev.langchain4j.internal.Utils.randomUUID;
 import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
 import dev.langchain4j.model.embedding.EmbeddingModel;
 import dev.langchain4j.store.embedding.EmbeddingStore;
-import dev.langchain4j.store.embedding.EmbeddingStoreWithoutMetadataIT;
-import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import io.weaviate.client.v1.schema.model.Property;
+import java.util.Arrays;
+import java.util.Collections;
+import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.weaviate.WeaviateContainer;
 
-import static dev.langchain4j.internal.Utils.randomUUID;
-
 @Testcontainers
-class LocalWeaviateEmbeddingStoreIT extends EmbeddingStoreWithoutMetadataIT {
+class LocalWeaviateEmbeddingStoreIT extends EmbeddingStoreIT {
 
     @Container
     static WeaviateContainer weaviate = new WeaviateContainer("semitechnologies/weaviate:latest")
             .withEnv("QUERY_DEFAULTS_LIMIT", "25")
             .withEnv("DEFAULT_VECTORIZER_MODULE", "none")
+            .withEnv("AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED", "true")
             .withEnv("CLUSTER_HOSTNAME", "node1");
 
-    EmbeddingStore<TextSegment> embeddingStore = WeaviateEmbeddingStore.builder()
+    private final EmbeddingStore<TextSegment> embeddingStore = WeaviateEmbeddingStore.builder()
             .scheme("http")
             .host(weaviate.getHost())
-            .port(weaviate.getFirstMappedPort())
+            .port(weaviate.getMappedPort(8080))
+            .useGrpc(true)
+            .grpcPort(weaviate.getMappedPort(50051))
             .objectClass("Test" + randomUUID().replace("-", ""))
+            .properties(Arrays.asList(new Property[]{
```

Comment:
It seems that dataType information is not used enywhere, can we get rid of it and simplify to this?

```java
.metadataKeys(List.of("string_empty", "string_space", ...))
```